javascript/modules/client/README.md

```diff
@@ -249,42 +271,50 @@ for await (
 
 ### Voting on an ERC20 proposal
 
-- __TODO__
+- **TODO**
 
 ### Loading the list of members (ERC20)
 
-- __TODO__
+Retrieving all the members of an ERC20 DAO.
+
+```ts
+import { ClientErc20 } from "@aragon/sdk-client";
+
+const client = new ClientERC20(context);
+const members = await client.methods.getMembers();
```

Comment:
We are not passing any kind of DAO/plugin address. If the DAO/plugin address is passed via the context, then we should show it. If it is a `getMembers` parameter, we should show it as well